src/main/java/de/tum/in/www1/artemis/service/StatisticsService.java

```diff
@@ -0,0 +1,95 @@
+package de.tum.in.www1.artemis.service;
+
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.repository.StatisticsRepository;
+
+@Service
+public class StatisticsService {
+
+    private final StatisticsRepository statisticsRepository;
+
+    public StatisticsService(StatisticsRepository statisticsRepository) {
+        this.statisticsRepository = statisticsRepository;
+    }
+
+    public Integer getLoggedInUsers(Long span) {
+        return this.statisticsRepository.getLoggedInUsers(ZonedDateTime.now().minusDays(span).toInstant());
+    }
+
+    public Integer getActiveUsers(Long span) {
+        return this.statisticsRepository.getActiveUsers(ZonedDateTime.now().minusDays(span));
+    }
+
+    /**
+     * Forwards the request to the repository, which returns a List<Map<String, Object>>, with String being the column name, "day" and "amount" and Object being the value,
+     * either the date or the amount of submissions. It then collects the amounts in an array, depending on the span value, and returns it
+     *
+     * @param span DAY,WEEK,MONTH or YEAR depending on the active tab in the view
+     * @return a array, containing the values for each bar in the graph
+     */
+    public Integer[] getTotalSubmissions(String span) {
```

Comment:
Why are you not using the Enum here?

Reply:
Good catch, [done](https://github.com/ls1intum/Artemis/pull/2471/commits/ea28fa7ce583b5250cde3499d6659b38eb254195)